examples/nodejs/with-typescript/main.ts

```diff
@@ -0,0 +1,30 @@
+// How to deploy a job using TypeScript.
+
+import { init } from 'dcp-client';
+
+const { compute } = await init();
+
+const inputSet = [1, 2, 3, 4];
+
+const workFunction = (input) => {
+  // Send a mandatory progress update to the scheduler.
```

Comment:
Suggestion - either put the comment on the same line as the progress call, or add a new line after the progress call.  For a first-time reader, this ties the comment only the code it affects more clearly.

Reply:
That makes sense! I've opted to add a blank line after the call to `progress`.